### PR TITLE
upgrade DQN's lr interface compatibility

### PR DESCRIPTION
**Pull request by rical730**
1. upgrade lr interface compatibility of dqn and ddqn
2. fix issue https://github.com/PaddlePaddle/PARL/issues/290

### CODE REVIEW

File: parl/algorithms/fluid/ddqn.py [hunk 1]
```diff
@@ -43,11 +41,29 @@ def __init__(
...
+              next_obs,
+              terminal,
+              learning_rate=None):
+        """ update value model self.model with DQN algorithm
+        """
+        # Support the modification of learning_rate
+        if learning_rate is None:
+            assert isinstance(
+                self.lr,
+                float), "Please pass parameter to lr (learning rate) of DQN."
```
Comment (TomorrowIsAnOtherDay) [inline parl/algorithms/fluid/ddqn.py]: Please set the learning rate of DQN in initializaion.

**Comment by TomorrowIsAnOtherDay** [comment]
Please update the DQN usage in the following example:

https://github.com/PaddlePaddle/PARL/blob/21ac9c69229e44fcdc010580712888b61b2ebdc6/examples/DQN/train.py#L83

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
Looks good to me.
